src/OrchardCore.Modules/OrchardCore.BackgroundTasks/Views/BackgroundTask/Edit.cshtml

```diff
@@ -44,6 +44,17 @@
                 </div>
             </div>
 
+            <div class="@Orchard.GetWrapperCssClasses()">
+                <div class="@Orchard.GetEndCssClasses(true)">
+                    <div class="form-check">
+                        <input type="checkbox" class="form-check-input" asp-for="UsePipeline" checked="@Model.UsePipeline" />
+                        <label class="form-check-label" asp-for="UsePipeline">@T["Use Tenant Pipeline"]</label>
+                    </div>
+                    <span class="hint">@T["Wether or not the tenant pipeline should be built and then executed."]</span>
+                    <span class="hint">@T["This to configure endpoints and then to allow route urls generation."]</span>
```

Comment:
I think you should combine the two hints into one that say
```
<span class="hint">@T["Deciding whether to construct and execute the tenant pipeline, which would subsequently allow you to generate URLs within the background task."]</span>
```

Reply:
For sure your english is better than mine ;)

But not sure that `to construct` is better than `should be built` meaning somewhere that nothing to do if already done, then maybe worth to mention `endpoints` for this quite advanced setting.

Yes, I used 2 hints to not have a too long code line (also for translations), knowing that at the end they are rendered inline one after each other.

But not fully sure myself so feel free to suggest comment tweaks in a separate PR.

Idem for the other suggestions on comments.